crd-generator/api/src/test/java/io/fabric8/crd/generator/ResourcesTest.java

```diff
@@ -42,14 +42,14 @@ public void shouldSupportMultiplePrinterColumns() {
   }
 
   @Test
-  public void shouldSupportMultipleSortPrinterColums() {
+  public void shouldSupportMultipleSortPrinterColumns() {
     Resources r = new Resources();
     SortPrinterColumnsDecorator dec1 = new SortPrinterColumnsDecorator("my-crd", "v1");
     SortPrinterColumnsDecorator dec2 = new SortPrinterColumnsDecorator("my-crd", "v2");
     r.decorate(dec1);
     r.decorate(dec2);
     assertEquals(2, r.getDecorators().size());
-    r.getDecorators().stream().forEach(d -> System.out.println(d));
+    r.getDecorators().forEach(System.out::println);
```

Comment:
It should be fine to start removing a log of the sys outs in tests.

Reply:
I removed the sys out, would be great if you could re-run the tests.